yrs/src/block.rs

```diff
@@ -244,9 +244,13 @@ impl BlockPtr {
                                         break;
                                     }
                                 } else {
-                                    if let Some(item_origin) = item.origin {
-                                        if items_before_origin.contains(&item_origin) {
```

Comment:
Initially we used `ID` for `items_before_origin` and `conflicting_items` while Yjs was using Item references. These are not the same (Item may be responsible for holding range of updates) as for items with squashed contents individual IDs may be not found, but the items are in there.